jenkins_github_checkwarn.sh

```diff
@@ -36,9 +35,9 @@ def_checkpatch_paths="${check_make}"
 if [ "${FULL_REVIEW}" -ne 0 ]; then
   # Skip pylint checking if performed by GitHub.
   if [ -e .github/workflows/pylint.yml ]; then
-    def_checkpatch_paths+=":${check_style}:${check_shell}"
+    def_checkpatch_paths+=":${check_shell}"
```

Comment:
Perhaps I should do something similar with clang-format that we do with pylint

Reply:
What do you mean?

Reply:
Like replicate line 37 and add check_style where clang-format isn't used.

Reply:
Oh, I see what you mean.  Yes, that would be safest with historic releases but given we have clang-format on master and 2.4 how many PRs will still use checkpatch?